PWGHF/correlationHF/AliHFAssociatedTrackCuts.cxx

```diff
@@ -475,6 +475,44 @@ Bool_t AliHFAssociatedTrackCuts::InvMassDstarRejection(AliAODRecoDecayHF2Prong*
 	
 	return kTRUE;
 }
+
+//________________________________________________________
+Bool_t AliHFAssociatedTrackCuts::InvMassSigmacRejection(AliAODRecoDecayHF3Prong* d, AliReducedParticle *redtrack, Int_t hypLc) const {
+	//
+	// Calculates invmass of track+D0 and rejects if compatible with D*
+	// (to remove pions from D*)
+	// 
+	Double_t nsigma = 3.;
+	Double_t mLc1, mLc2;
+        mLc1 = d->InvMassLcpKpi();
+        mLc2 = d->InvMassLcpiKp();	
+	
+	Double_t invmassSigmac1 = 0, invmassSigmac2 = 0; 
+	Double_t e1P = d->EProng(0,2212), e2K = d->EProng(1,321), e3Pi = d->EProng(2,211); //hyp 1 (p,K,pi) - Lc1
+	Double_t e1Pi = d->EProng(0,211), e3P = d->EProng(2,2212); //hyp 2 (pi, K, p) - Lc2
+	Double_t psum2 = (d->Px()+redtrack->Px())*(d->Px()+redtrack->Px())
+	+(d->Py()+redtrack->Py())*(d->Py()+redtrack->Py())
+	+(d->Pz()+redtrack->Pz())*(d->Pz()+redtrack->Pz());
+
+	switch(hypLc) {
+		case 1:
+			invmassSigmac1 = TMath::Sqrt(pow(e1P+e2K+e3Pi+redtrack->Epion(),2.)-psum2);
+			if ((TMath::Abs(invmassSigmac1-mLc1)-0.1669) < nsigma*1.*pow(10.,-3.)) return kFALSE;
+			break;
```

Comment:
Not sure about the 1.* pow(10.,-3.)
I see from the sigma paper that they use a Voigt funciton, which is a convolution of a Gaussian (with sigma 1 MeV) and Breit-Wigner (with sigma 1.83-1.89 MeV for Sigmac^0 and Sigmac^++).
Despite the Voigt doesn't have a proper 'sigma' definition, using 1 MeV would be an underestimation (indeed the full extension of the function from the mass plot in the paper is of about 10 MeV). I'd stay, thus, 2ith a larger value, like 1.5*pow(10.,-3.), which gives a 9 MeV extension, left-to-right, which is safe enough.